lib/util.js

```diff
@@ -579,16 +603,12 @@ module.exports = {
       };
     }
 
-    _.each([APP_JSON, APP_XML, APP_VND_JSON], (supportedCType) => {
-      // these are the content-types that we'd prefer to generate a body for
-      // in this order
-      if (contentObj[supportedCType]) {
-        cTypeHeader = supportedCType;
+    for (let cType in contentObj) {
```

Comment:
https://stackoverflow.com/questions/135448/how-do-i-check-if-an-object-has-a-property-in-javascript/136411#136411